cookbooks/burnman/doc/burnman.tex

```diff
@@ -15,25 +16,28 @@ \subsubsection{2D compressible convection with a reference profile and material
 to generate the reference profile has the advantage that all the material property data
 are consistent, for example, the gravity profile is computed using the reference density.
 \begin{figure}
-  \includegraphics[width=\textwidth]{./reference_profile.pdf}
+  \includegraphics[width=\textwidth]{../../cookbooks/burnman/doc/reference_profile.pdf}
```

Comment:
I think you can also write this and all other places as 
```suggestion
  \includegraphics[width=\textwidth]{cookbooks/burnman/doc/reference_profile.pdf}
```

Can you try?

Reply:
Yes, I will try that and also remove some past commits. I was just wondering that the path is relative to the manual folder, so if we remove the linked list, this path would not work, or am I understanding it incorrectly?